kaggle_environments/agent.py

```diff
@@ -34,15 +34,21 @@ def is_url(url):
         return False
 
 
-def get_last_callable(raw, fallback=None):
+def get_last_callable(raw, fallback=None, path=None):
     orig_out = sys.stdout
     buffer = StringIO()
     sys.stdout = buffer
 
     try:
-        code_object = compile(raw, "<string>", "exec")
+        code_object = compile(raw, path, "exec")
```

Comment:
@harrisse more unfamiliar with this file, do these changes look good here?

Reply:
Besides the configuration line below, yes, these are nice updates, thanks Stone :)